model/clusters_mgmt/v1/cluster_operators_resource.model

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright (c) 2019 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Provides detailed information about the Metrics of the Cluster
+resource ClusterOperators {
```

Comment:
For consistency with the other "metric queries" I think that this should be renamed to "ClusterOperatorsMetricQuery".